packages/esbuild/BUILD.bazel

```diff
@@ -60,14 +60,29 @@ filegroup(
         "esbuild.bzl",
         "esbuild_config.bzl",
         "esbuild_packages.bzl",
-        "esbuild_repositories.bzl",
+        # Intentionally not shipping the repositories code via npm
+        # See comment below on :builtin_srcs
+        # "esbuild_repositories.bzl",
```

Comment:
Can `esbuild_packages` also be removed from here?

Reply:
no, it's still loaded from toolchain/BUILD.bazel